gaps.exceptions.gapsIOError
===========================

.. currentmodule:: gaps.exceptions

.. autoexception:: gapsIOError